etc/config.in

```diff
@@ -66,16 +66,24 @@
 # 4. $DAPSROOT/etc/config
 
 
-## Key:         ASPELL_EXTRA_DICT
+## Key:         SPELL_CHECKER
+## ----------------------
+## Description: Spellchecker application to use
+## Type:        Name of the spellchecker's binary
+## Default:     "aspell"
+#
+SPELL_CHECKER="aspell"
```

Comment:
We can probably solve this in a more elegant way by looking for aspell and hunspell on the system and decide which one to use based on what is installed. If we do so, this would rather look like

        ## Key:         SPELL_CHECKER
        ￼## ----------------------
        ￼## Description: Spellchecker application to use
        ￼## Type:        Name of the spellchecker's binary
        ## Default:     DAPS performa a lookup for aspell and hunspell. If only one     
        ￼##                  of these packages is installed, it will be used automatically. If
        ##                  both are installed, aspell will be used.
        ##                  If you want to force using one of the two spellcheckers, specify
        ##                  it here.
        ￼SPELL_CHECKER=""

See below for more code...